_posts/research/subteams/2022-01-31-learning-modeling-robotics.md

```diff
@@ -86,6 +86,24 @@ Specifically, we are studying how reinforcement learning might be applied to aut
 
 {% include image.html url="research/isat.jpg" description="iSAT's vision for student-AI teaming and classroom orchestration." %}
 
+## 1.5. Hierarchical Structures for Human-Agent Interaction
```

Comment:
Move it up. This is more relevant than other things just because we are having papers on it. The most recent article/pub should be on top and the rest should be reverse chronological order.

Reply:
Also, I think we should have this in the other strand. This is more human-AI teaming than learning for robotics.

Reply:
Move it up more -- this should be 1.1 as it's the last work and the first one in reverse chronological order